Specs/Scene/Model/MaterialPipelineStageSpec.js

```diff
@@ -19,6 +19,7 @@ import {
   ShaderBuilder,
 } from "../../../Source/Cesium.js";
 import createScene from "../../createScene.js";
+import ShaderBuilderTester from "../../ShaderBuilderTester.js";
```

Comment:
`"adds material uniforms"` and `"adds metallic roughness uniforms"` test basically the same thing: the only thing that's different are the `expectedUniforms`, which are subsets of the larger uniform map. let's combine them so they're not redundant